src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -35,6 +87,7 @@ public class HelpWindow extends UiPart<Stage> {
     public HelpWindow(Stage root) {
         super(FXML, root);
         helpMessage.setText(HELP_MESSAGE);
+        setHelpText();
```

Comment:
Maybe some UI testing is needed. Otherwise great job!! I am sure this will be useful for our users